config/config.json

```diff
@@ -30,8 +30,14 @@
       "vault_transit_path": "transit"
     }
   },
-  "VAULT_CACERT": "config/myCA.crt",
-  "VAULT_MTLS_CLIENT_CERT": "config/mtls_auth.crt",
-  "VAULT_MTLS_CLIENT_KEY": "config/mtls_auth.key",
-  "VAULT_URL": "https://vault:8300"
+  "VAULT": {
+    "cacert": "config/myCA.crt",
+    "mtls_client_cert": "config/mtls_auth.crt",
+    "mtls_client_key": "config/mtls_auth.key",
+    "url": "https://vault:8300",
+    "namespace": "",
```

Comment:
@in0rdr please note that namespace is empty and might require `tenant` as value as for now

Reply:
Yes, I noticed that the empty value is interpreted as "not set":
```
[2021-12-20 15:43:29,785] distributey {/home/andi/Downloads/distributey/distributey/vault_backend.py:61} WARNING - tenant: monitoring, origin: 127.0.0.1, ua: curl/7.68.0 - Cannot find dedicated Vault namespace for tenant "monitoring". Using global default.
```

I think this is not the desired behavior. If we supply an empty namespace/tenant (""), we really want Distributey to lookup the Root namespace and not fallback to the global defaults.

Reply:
> If we supply an empty namespace/tenant (""), we really want Distributey to lookup the Root namespace and not fallback to the global defaults.

That's actually a 🐞 that you found! The problem is that `""` should signalize to connect to the root namespace. However, Python treats empty strings as `False`, which is why this value gets overwritten by the global default, which is of course not intended. I will add a quick fix that requires `root` as value for root namespaces.